nautobot/docs/installation/http-server.md

```diff
@@ -14,16 +14,21 @@ be installed on your Nautobot server in a secure location that is readable only
 
 Two files will be created: the public certificate (`nautobot.crt`) and the private key (`nautobot.key`). The certificate is published to the world, whereas the private key must be kept secret at all times.
 
-!!! info
-    Some Linux installations, including CentOS, have changed the location for SSL certificates from `/etc/ssl/` to `/etc/pki/tls/`. The command below may need to be changed to reflect the certificate location.
+=== "Ubuntu/Debian"
 
-    The following command will prompt you for additional details of the certificate; all of which are optional.
+    ```no-highlight
+    sudo openssl req -x509 -nodes -days 365 -newkey rsa:2048 \
+    -keyout /etc/ssl/private/nautobot.key \
+    -out /etc/ssl/certs/nautobot.crt
+    ```
```

Comment:
Should this be?


```suggestion
    ```no-highlight
    sudo openssl req -x509 -nodes -days 365 -newkey rsa:2048 \
      -keyout /etc/ssl/private/nautobot.key \
      -out /etc/ssl/certs/nautobot.crt
    ```
```

Reply:
I think both would have been fine, with this being the more complete convention.